vault-core/src/main/java/org/apache/jackrabbit/vault/fs/config/DefaultWorkspaceFilter.java

```diff
@@ -241,6 +241,10 @@ public void setSkipFilterChecksOnImport(boolean skipFilterChecksOnImport) {
         this.skipFilterChecksOnImport = skipFilterChecksOnImport;
     }
 
+    public boolean getSkipFilterChecksOnImport() {
```

Comment:
If this only affects import I would rather move it to `ImportOptions` and also clarify that this only affects checks against package items not about existing repo items.

Reply:
``ImportOptions`` already has such a method; i clarified the javadoc as recommended

Reply:
No need to add new methods/fields to WorkspaceFilter then, just use the one from ImportOptions directly (i.e. pass as boolean argument to `DocViewImporter`

Reply:
I went through and changed them ... a bit less changes than I actually anticipated.